packages/block-library/src/latest-posts/edit.js

```diff
@@ -466,7 +483,11 @@ export default function LatestPostsEdit( { attributes, setAttributes } ) {
 								.join( ' ' ) }
 							{ /* translators: excerpt truncation character, default …  */ }
 							{ __( ' … ' ) }
-							<a href={ post.link } rel="noopener noreferrer">
+							<a
```

Comment:
In other places, we don't use real links at all. Example:

https://github.com/WordPress/gutenberg/blob/a50c7178da24350956b52357be40c94d36a8b803/packages/block-library/src/comment-reply-link/edit.js#L49

Reply:
It's been discussed thoroughly to find the best accessible solution for this and we'll probably [replace this pseudo-links](https://github.com/WordPress/gutenberg/pull/40593#issuecomment-1109509018) with the  approach of this PR.